cmd/bcpd/Makefile

```diff
@@ -1,25 +1,18 @@
-.PHONY: all install build test cover deps tools prototools protoc clean dist
+.PHONY: all build test image tf protoc clean dist
```

Comment:
nice an cleaner... why did you pull out tf, btw?
i guess you need to `make tf` in the command dir now, not in root level.

Reply:
The test code need tendermint installed to run properly.

https://github.com/iov-one/weave/blob/merge-bcp-demo/cmd/bcpd/commands/start_test.go#L59-L63